plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-list-item/payment-gateway-list-item.tsx

```diff
@@ -32,83 +34,100 @@ export const PaymentGatewayListItem = ( {
-							'woocommerce'
+	return (
+		<div
+			id={ gateway.id }
+			className={ `transitions-disabled woocommerce-list__item woocommerce-list__item-enter-done woocommerce-item__payment-gateway ${
+				isWcPay ?? `woocommerce-item__woocommerce-payment`
+			} ${ hasIncentive ?? `has-incentive` }` }
+			{ ...props }
+		>
+			<div className="woocommerce-list__item-inner">
+				<div className="woocommerce-list__item-before">
+					<DefaultDragHandle />
+					<img src={ gateway.icon } alt={ gateway.title + ' logo' } />
+				</div>
+				<div className="woocommerce-list__item-text">
+					<span className="woocommerce-list__item-title">
+						{ gateway.title }
+						{ hasIncentive ? (
+							<StatusBadge
+								status="has_incentive"
+								message={ __(
+									'Save 10% on processing fees',
+									'woocommerce'
+								) }
+							/>
+						) : (
+							<StatusBadge status={ determineGatewayStatus() } />
+						) }
+					</span>
+					<span
+						className="woocommerce-list__item-content"
+						dangerouslySetInnerHTML={ sanitizeHTML(
+							decodeEntities( gateway.description )
 						) }
 					/>
-				) : (
-					<StatusBadge status={ determineGatewayStatus() } />
-				) }
-			</>
-		),
-		content: (
-			<>
-				<span
-					dangerouslySetInnerHTML={ sanitizeHTML(
-						decodeEntities( gateway.description )
+					{ isWcPay && (
+						<WooPaymentMethodsLogos
+							maxElements={ 10 }
+							isWooPayEligible={ true }
+						/>
 					) }
-				/>
-				{ isWCPay && (
-					<WooPaymentMethodsLogos
-						maxElements={ 10 }
-						isWooPayEligible={ true }
-					/>
-				) }
-			</>
-		),
-		after: (
-			<div className="woocommerce-list__item-after__actions">
-				<>
-					<PaymentGatewayButtons
-						id={ gateway.id }
-						isOffline={ false }
-						enabled={ gateway.state?.enabled || false }
-						needsSetup={ gateway.state?.needs_setup }
-						testMode={ gateway.state?.test_mode }
-						settingsUrl={ gateway.management?.settings_url || '' }
-					/>
-					<EllipsisMenu
-						label={ __( 'Task List Options', 'woocommerce' ) }
-						renderContent={ ( { onToggle } ) => (
-							<EllipsisMenuContent
-								pluginId={ gateway.id }
-								pluginName={ gateway.plugin.slug }
-								isSuggestion={ false }
-								links={ gateway.links }
-								onToggle={ onToggle }
-								isWooPayments={ isWCPay }
-								isEnabled={ gateway.state?.enabled }
+				</div>
+				<div className="woocommerce-list__item-after">
+					<div className="woocommerce-list__item-after__actions">
+						<>
+							<PaymentGatewayButtons
+								id={ gateway.id }
+								isOffline={ false }
+								enabled={ gateway.state?.enabled || false }
 								needsSetup={ gateway.state?.needs_setup }
 								testMode={ gateway.state?.test_mode }
+								settingsUrl={
+									gateway.management?.settings_url || ''
+								}
 							/>
-						) }
-					/>
-				</>
+							<EllipsisMenu
+								label={ __(
+									'Task List Options',
```

Comment:
```suggestion
									'Payment Provider Options',
```

Reply:
I'll fix it in my PR! https://github.com/woocommerce/woocommerce/pull/53590

Reply:
done https://github.com/woocommerce/woocommerce/pull/53590/commits/89be2c71fa9da684a938d1f0a53dce141e6721c9